app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+
+  def self.update_comments_counter(post_id)
+    post = Post.find(post_id)
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ]  Good job, for the association between post and comment, but there should be another association between comment and the user , for the comment belong to the user as well.  Please add the association as well for your app to work properly.

- [OPTIONAL ]  Your update_comment_counter method could be improved, to reduce it's complexity and the user of `postId `, by just  considering that You have the Post class in your object scope(ActiveRecord ) and use` increment!`  ActiveRecord method that increments the attribute by 1 and saves the record.